app/views/companies/show.html.erb

```diff
@@ -1,2 +1,2 @@
-<h1>Companies#show</h1>
-<p>Find me in app/views/companies/show.html.erb</p>
+<h1><%= @company.name %></h1>
+<%= link_to "back", :companies %>
```

Comment:
@jrpatton1991 @kaytho @romaniapm this line will work
Keep in mind you can also `companies_path` in place of `:companies`

If you need to see what routes you have, just run `rake routes` in your terminal